ironfish-cli/src/utils/fees.ts

```diff
@@ -20,7 +20,7 @@ import { promptCurrency } from './currency'
 export async function selectFee(options: {
   client: Pick<RpcClient, 'wallet'>
   transaction: CreateTransactionRequest
-  account?: string
+  account: string | undefined
```

Comment:
this is effectively the same thing right?

Reply:
Not quite, with `string | undefined` and no `?`, the developer is required to specify account in the object (at least in the ways that we use this function). I added this change because I've seen this bug pretty much every time we've used `selectFee`: https://github.com/iron-fish/ironfish/pull/3919